README.md

```diff
@@ -41,6 +41,8 @@ This project was created using ASP.NET Core 6.0. This application aims to demons
   
   3. Finally, run the application using the command `dotnet run`. After the application has started, it will display a URL in the `command line interface`, typically something like (e.g., https://localhost:5001). Copy this URL and paste it into your default web browser.
 
+> **NOTE:** By default, the dashboard listing is hidden in the sample. If you want to render a dashboard with a specific listing of dashboards, you need to add a suburl to the web address and retrieve the dashboard listing accordingly.(e.g., https://localhost:5001/dashboardlisting).
```

Comment:
We represent the dashboard embedding by default without the dashboards listing sidebar. You must navigate to the "dashboardlisting" URL (such as https://localhost:5001/dashboardlisting) to enable the dashboards list.

Reply:
Update the port number and URL for other two platform samples.

Reply:
Content updated